src/components/cylc/Toolbar.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <v-toolbar
+    flat
+    dense
+    class="c-toolbar"
+  >
+    <div class="v-toolbar-title">
+      <v-toolbar-title
+        class="tertiary--text font-weight-light"
+      >
+        <!-- burger button for mobile -->
+        <v-btn
+          v-if="responsive"
+          class="default v-btn--simple"
+          dark
+          icon
+          @click.stop="onClickBtn"
+        >
+          <v-icon>mdi-view-list</v-icon>
+        </v-btn>
+        <!-- title -->
+        <span class="c-toolbar-title">{{ title }}</span>
+      </v-toolbar-title>
+    </div>
+
+    <!-- control bar elements displayed only when a suite has been positioned -->
+    <template v-if="tree && tree.length > 0">
+      <a @click="onClickPause">
+        <v-icon color="#5E5E5E">mdi-pause</v-icon>
+      </a>
+
+      <a @click="onClickStop">
+        <v-icon color="#5E5E5E">mdi-stop</v-icon>
+      </a>
+
+      <a>
+        <v-chip color="#E7E7E7" @click="toggleExtended">Control</v-chip>
+      </a>
+
+      <span>Running, will stop at 30000101T0000 cycle</span>
+
+      <v-spacer />
+
+      <a class="add-view" @click="onClickAddView">
+        Add View <v-icon color="#5995EB">mdi-plus-circle</v-icon>
+      </a>
+    </template>
+
+    <!-- displayed only when extended===true -->
+    <template v-slot:extension v-if="extended">
+      <span style="margin-left: 260px;">
+        <a @click="onClickPause">
```

Comment:
The `control` button is supposed to replace the functionality of the current `control` menu so will have things like `poll` and the full range of `stop` options, etc.

We can add these options in a follow-up.